charts/agent-control-deployment/tests/configmap_agentcontrol_config_test.yaml

```diff
@@ -75,6 +75,8 @@ tests:
                 token_url: https://system-identity-oauth.service.newrelic.com/oauth2/token
               endpoint: https://opamp.service.newrelic.com/v1/opamp
               fleet_id: abcefg
+              signature_validation:
```

Comment:
if not specified signature_validation is enabled=true by default, right?

Reply:
I removed the possibility to make it configurable from the `values.yaml`. So the endpoint is always added.